### ISSUE TITLE
help: problem creating a service?????

### ISSUE THREAD

**Issue by sebastianovide** [Swiip/generator-gulp-angular#515]
Hello all,

I've created few services without problems, and now, creating a new empty service, angular complains that he cannot find the module ??? 

The problematic service has the usual format and is located in a subfolder src/app/subfolder

```
'use strict';

angular
  .module('dashboard')
  .factory('MyService', function($q) {
   return {
....
....
....
   };
   });
```

the error that I'm getting is

` Uncaught Error: [$injector:nomod] Module 'dashboard' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.`

any ideas ?

**update**
Moving the file to src/app fixed the problem.... which forces me to put all the services under the same folder (????) instead of keeping them organised by functional areas.

**Comment by vredchenko** [comment]
Hi.

I'm creating my factories/services in subfolders (under src/app/whatever). Here's what one of them looks like:

```
'use strict';
//ngInject
function CompetenciesFactory() {
  return {
    foo: 'bar'
  };
}
export default CompetenciesFactory;
```

Then in index.js:

```
'use strict';

import NavbarCtrl from './components/navbar/navbar.controller';
import LandingCtrl from './landing/landing.controller';
import InterviewCtrl from './interview/interview.controller';
import CompetenciesFactory from './interview/webdev-competencies.factory';

angular.module('interviewReport', ['ngAnimate', 'ngCookies', 'ngTouch', 'ngSanitize', 'ngResource', 'ui.router', 'mgcrea.ngStrap'])
  .controller('LandingCtrl', LandingCtrl)
  .controller('InterviewCtrl', InterviewCtrl)
  .controller('ReportCtrl', ReportCtrl)
  .controller('NavbarCtrl', NavbarCtrl)
  .factory('CompetenciesFactory', CompetenciesFactory)
// etc.
```
Hope that answers your question

**Comment by jbaroudi** [comment]
Hi @sebastianovide i believe the problem is that you need to set 

**the module**

`angular.module( 'dashboard')`

**with**

`angular.module( 'dashboard', [ ] ).....`

to declare the module (missing brackets)

and then, add the reference in your app/index.js 

hope this solves your problem

**Comment by zckrs** [comment]
:tada: Just released v0.12.0 :tada:
Your issue should be fixed. I close but if you can retry your case with a new fresh install of generator you are welcome. Any feedback would be appreciated (positive comments too ^_^).
